doc/source/man.rst

```diff
@@ -327,52 +325,52 @@ A scan can be run by specifying the profile to use and where to write the CSV fi
+The following options are available for every Rho command.
 
 ``--help``
 
-  This prints the help for the rho command or subcommand.
+  Prints the help for the ``rho`` command or subcommand.
 
 ``-v``
 
-  The verbose mode (``-vvv`` for more, ``-vvvv`` to enable connection debugging).
+  Enables the verbose mode. The ``-vvv`` option increases verbosity to show more information. The ``-vvvv`` option enables connection debugging.
 
 Examples
 --------
 
-:Adding new authentication credentials with a keyfile: ``rho auth add --name=new-creds --username=rho-user --sshkeyfile=/etc/ssh/ssh_host_rsa_key``
-:Adding new authentication credentials with a password: ``rho auth add --name=other-creds --username=rho-user-pass --password``
-:Creating a new profile: ``rho profile add --name=new-profile --hosts 1.2.3.0 --auth new-creds``
-:Editing a profile: ``rho profile edit --name=new-profile --hosts 1.2.3.[0:255] --auth new-creds other-creds``
+:Creating a new authentication profile with a keyfile: ``rho auth add --name=new-creds --username=rho-user --sshkeyfile=/etc/ssh/ssh_host_rsa_key``
+:Creating a new authentication profile with a password: ``rho auth add --name=other-creds --username=rho-user-pass --password``
+:Creating a new profile: ``rho profile add --name=new-profile --hosts 1.192.0.19 --auth new-creds``
+:Editing a profile: ``rho profile edit --name=new-profile --hosts 1.192.0.[0:255] --auth new-creds other-creds``
 :Running a scan with a profile: ``rho scan --profile=new-profile --reportfile=/home/jsmith/Desktop/output.csv``
 
 Security Considerations
 -----------------------
 
-The credentials used to access servers are stored with the profile configuration in an AES-256 encrypted configuration file. A vault password is used to access this file. The vault password and decrypted file contents are in the system memory, and could theoretically be written to disk if they were to be swapped out.
+The authentication profile credentials that are used to access servers are stored with the network profile configuration in an AES-256 encrypted configuration file. A vault password is used to access this file. The vault password and decrypted file contents are in the system memory, and could theoretically be written to disk if memory swapping is enabled.
 
-While the vault password can be passed via a file to run ``rho`` without prompts (such as scheduling a cron job), using this can be risky and should be stored in a location with limited access; be cautious about using this mechanism.
+Although you can run the ``rho`` command without prompts (such as scheduling a cron job) by using a file to pass the vault password, the use of a file for vault password storage is not without risk; therefore, its use requires caution. The vault password allows access to encrypted Rho data that could contain sensitive information. Make sure that this vault password file, if used, is stored in a location that has limited access.
```

Comment:
`s/Rho/rho/` again

Reply:
Thank you for making these comments, even though I decided I disagreed a bit. It made me go back and check all the references.